sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClientTest.java

```diff
@@ -122,6 +130,43 @@ void teardown() {
         consumer.close();
     }
 
+    /**
+     * Verifies that this peek one messages.
+     */
+    @Test
+    void peekOneMessage() {
+        // Arrange
+        final int numberOfEvents = 1;
+        when(managementNode.peek())
+            .thenReturn(Mono.just(mock(ServiceBusReceivedMessage.class)));
+
+        // Act & Assert
+        StepVerifier.create(consumer.peek())
```

Comment:
Do you have a test that peeks the next sequence number?

Reply:
added this test